wiki/Beatmap_Editor/en.md

```diff
@@ -1,13 +1,35 @@
 # Beatmap Editor
 
-The **beatmap editor** is osu!'s in-game graphical interface for creating [beatmaps](/wiki/Beatmaps). It is separated into four parts:
+The **beatmap editor** is osu!'s in-game graphical interface for creating [beatmaps](/wiki/Beatmaps). This is where all [beatmapping](/wiki/Beatmapping) takes place, and all content for the game is created. Creating content for osu! is, and always will be, free. So feel free to jump right in and start beatmapping!
```

Comment:
second to last sentence doesn't belong here, and the last sentence is too imperative (this article isn't meant to encourage people to do anything). as long as u can hit Beatmaps and Beatmapping in this first part and give some very minimal context for each that's probably good. something like the intro of https://osu.ppy.sh/help/wiki/Modding

also "all content for the game" isnt very explanatory and kinda misleading